src/reduxTypes.js

```diff
@@ -266,6 +266,7 @@ export type RealmState = $ReadOnly<{|
   twentyFourHourTime: boolean,
   canCreateStreams: boolean,
   isAdmin: boolean,
+  mandatoryTopics: boolean,
```

Comment:
nit: this goes in the previous group in this definition:
https://github.com/zulip/zulip-mobile/pull/4798#discussion_r652275167